selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/provider/CheTestWorkspaceAgentApiEndpointUrlProvider.java

```diff
@@ -48,4 +48,14 @@ public URL get(String workspaceId) throws Exception {
     }
     throw new RuntimeException("Cannot find dev machine on workspace with id " + workspaceId);
   }
+
+  /**
+   * Checks whether provided {@link Machine} contains wsagent server.
+   *
+   * @param machine machine to check
+   * @return true when wsagent server is found in provided machine, false otherwise
+   */
+  public static boolean containsWsAgentServer(Machine machine) {
+    return machine.getServers().keySet().contains(Constants.SERVER_WS_AGENT_HTTP_REFERENCE);
```

Comment:
Actually it's a good time to get rid of `Constants.SERVER_WS_AGENT_HTTP_REFERENCE` as well.

P.S. Along with other unused constants like `SERVER_WS_AGENT_WEBSOCKET_REFERENCE`, `SERVER_EXEC_AGENT_WEBSOCKET_REFERENCE`, `WS_AGENT_PORT`